### DISCUSSION TITLE
Creating a unordered list items

### DISCUSSION THREAD

**Discussion by ghost** [handlebars-lang/handlebars.js#1930] [Q&A]
I have a handlebar html

`<ul><li>{{#listConstruct product}}{{this}}{{/listConstruct}}</li></ul>`

Need to have output like this 

```
<ul>
<li>Pen</li>
<li>Pencil</li>
<li>Notebook</li>
</ul>
```
I wrote a listConstruct helper like this.
 ```
Handlebars.registerHelper('listConstruct', function(items) {
   let res = '';
    items.forEach((val)=>{res+= '<li>'+val+'</li>'});
   return res;
 });
```

But this result in output like this 

`<ul><li><li>Pen</li><li>Pencil</li><li>Notebook</li></li></ul>`

How to remove the extra li inside of UL.

**Comment by jaylinski** [comment]
There is an example for this in the readme:

* https://handlebarsjs.com/guide/builtin-helpers.html#each
* https://handlebarsjs.com/examples/builtin-helper-each-block.html
